.github/workflows/docker-translation.yml

```
@@ -18,12 +18,14 @@ on:
 
 jobs:
   get-environment:
+    if: github.event.pull_request.base.repo.name  == 'centreon'
     uses: ./.github/workflows/get-environment.yml
     with:
       version_file: centreon/www/install/insertBaseConf.sql
 
   dockerize:
     needs: [get-environment]
+    if: needs.get-version.outputs.pull_request_environment == 'internal_contributor'
```

actionlint

property "get-version" is not defined in object type {get-environment: {outputs: {is_cloud: string; is_nightly: string; is_targeting_feature_branch: string; latest_major_version: string; major_version: string; minor_version: string; os_and_database_matrix: string; release: string; release_type: string; stability: string; target_stability: string}; result: string}}

(expression)